heltour/tournament/forms.py

```diff
@@ -46,9 +47,11 @@ def __init__(self, *args, rules_url='', **kwargs):
         self.season = kwargs.pop('season')
         
         self.username = kwargs.pop('user')
-        
+
+        self.player, created = Player.objects.get_or_create(lichess_username=self.username)
```

Comment:
Won't this create and use a player with an empty username when the form is initially instantiated?

Pretty sure it will

Reply:
Also, why are we using  a `username` field in this case. Can't we enforce a Player object to be in scope because you've already signed in?